src/storage/index/hash_index.cpp

```diff
@@ -208,6 +220,7 @@ void HashIndex<T>::insertIntoPersistentIndex(Key key, offset_t value) {
         ;
     copyKVOrEntryToSlot<Key, false /* insert kv */>(
         iter.slotInfo, iter.slot, key, value, fingerprint);
+    updateSlot(iter.slotInfo, iter.slot);
```

Comment:
Why do `updateSlot` here again? I thought you already did that inside `copyKVOrEntryToSlot`.

Reply:
I seem to recall removing that part of `copyKVOrEntryToSlot`, but I must have reverted it. I've fixed it (and merged the first two arguments since it's only being used with an iterator).